lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +18,17 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(text, str):
```

Comment:
revert the condition and return None immediately

Reply:
done